src/CoreWCF.NetTcp/src/CoreWCF/MessageSecurityOverTcp.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using CoreWCF.Channels;
+using CoreWCF.Runtime;
+using CoreWCF.Security;
+
+namespace CoreWCF
+{
+    public sealed class MessageSecurityOverTcp
+    {
+        internal const MessageCredentialType DefaultClientCredentialType = MessageCredentialType.Windows;
+        internal const string DefaultServerIssuedTransitionTokenLifetimeString = "00:15:00";
+
+        MessageCredentialType _clientCredentialType;
+        SecurityAlgorithmSuite _algorithmSuite;
```

Comment:
Missing accessors. wasAlgorithmSuiteSet should begin with _

Reply:
done